### ISSUE TITLE
I/HwPointEventFilter: do not support AFT because of no config

### ISSUE THREAD

**Issue by camillo860** [TakuSemba/RtmpPublisher#5]
Hello, 
I try your RtmpPublisher.
I set streaming_url and i check permission camera and microphono.
The app crash, this isa report of debug. I use Huawei P9 Lite with Android 7.0

`
04/08 23:22:18: Launching app
$ adb shell am start -n "com.takusemba.rtmppublishersample/com.takusemba.rtmppublishersample.MainActivity" -a android.intent.action.MAIN -c android.intent.category.LAUNCHER -D
Waiting for application to come online: com.takusemba.rtmppublishersample.test | com.takusemba.rtmppublishersample
Waiting for application to come online: com.takusemba.rtmppublishersample.test | com.takusemba.rtmppublishersample
Waiting for application to come online: com.takusemba.rtmppublishersample.test | com.takusemba.rtmppublishersample
Connecting to com.takusemba.rtmppublishersample
Capturing and displaying logcat messages from application. This behavior can be disabled in the "Logcat output" section of the "Debugger" settings page.
I/System.out: Sending WAIT chunk
I/art: Debugger is active
Connected to the target VM, address: 'localhost:8605', transport: 'socket'
I/System.out: Debugger has connected
I/System.out: waiting for debugger to settle...
I/System.out: waiting for debugger to settle...
I/System.out: waiting for debugger to settle...
I/System.out: waiting for debugger to settle...
I/System.out: waiting for debugger to settle...
I/System.out: waiting for debugger to settle...
I/System.out: debugger has settled (1336)
I/InstantRun: starting instant run server: is main process
I/HwCust: Constructor found for class android.app.HwCustActivityImpl
I/HwCust: Constructor found for class android.app.HwCustHwWallpaperManagerImpl
W/art: Before Android 4.1, method android.graphics.PorterDuffColorFilter android.support.graphics.drawable.VectorDrawableCompat.updateTintFilter(android.graphics.PorterDuffColorFilter, android.content.res.ColorStateList, android.graphics.PorterDuff$Mode) would have incorrectly overridden the package-private method in android.graphics.drawable.Drawable
E/HW-JPEG-DEC: [HME_JPEG_DEC_Delete](3321): HME_JPEG_DEC_Delete: decoder_ctx=null
E/Camera: Camera new cameraInitNormal:0
I/HwSecImmHelper: mSecurityInputMethodService is null
I/HwPointEventFilter: do not support AFT because of no config
I/OpenGLRenderer: Initialized EGL, version 1.4
W/linker: /vendor/lib64/libhwuibp.so: unused DT entry: type 0xf arg 0xe3a
W/art: Before Android 4.1, method int android.support.v7.widget.ListViewCompat.lookForSelectablePosition(int, boolean) would have incorrectly overridden the package-private method in android.widget.ListView
E/OpenGLRenderer: allen debug liyu Key: 0
E/OpenGLRenderer: allen debug liyu Key: 1
E/OpenGLRenderer: allen debug liyu Key: 240518168576
E/OpenGLRenderer: allen debug liyu Key: 68719476736
E/OpenGLRenderer: allen debug liyu Key: 34359738371
W/IInputConnectionWrapper: reportFullscreenMode on inexistent InputConnection
E/Camera: Camera new cameraInitNormal:0
I/art: Do partial code cache collection, code=30KB, data=25KB
I/art: After code cache collection, code=30KB, data=25KB
I/art: Increasing code cache capacity to 128KB

       [ 04-08 23:22:33.716   544:31475 E/         ]
       void hw::COutMetaSplitBase::dump_thermal_meter(monitor_t): open  failed!
I/hwaps: JNI_OnLoad
I/OMXClient: MuxOMX ctor
I/OMXClient: MuxOMX ctor
I/ACodec: In onAllocateComponent create compenent, codec name: OMX.IMG.TOPAZ.VIDEO.Encoder
I/ACodec: In onAllocateComponent create compenent, codec name: OMX.google.aac.encoder
W/ACodec: do not know color format 0x7f000001 = 2130706433
W/ACodec: do not know color format 0x7f000005 = 2130706437
W/ACodec: do not know color format 0x7f000002 = 2130706434
W/ACodec: do not know color format 0x7f000006 = 2130706438
W/ACodec: do not know color format 0x17 = 23
W/ACodec: do not know color format 0x18 = 24
W/ACodec: do not know color format 0x7f000011 = 2130706449
W/ACodec: do not know color format 0x16 = 22
W/ACodec: do not know color format 0x7f000007 = 2130706439
W/ACodec: do not know color format 0x7f000008 = 2130706440
W/ACodec: do not know color format 0x19 = 25
W/ACodec: do not know color format 0x1a = 26
W/ACodec: do not know color format 0x1b = 27
W/ACodec: do not know color format 0x1c = 28
W/ACodec: do not know color format 0x7f000003 = 2130706435
W/ACodec: do not know color format 0x7f000009 = 2130706441
W/ACodec: do not know color format 0x7f00000c = 2130706444
W/ACodec: do not know color format 0x7f000012 = 2130706450
W/ACodec: do not know color format 0x7f00000a = 2130706442
W/ACodec: do not know color format 0x7f00000b = 2130706443
W/ACodec: do not know color format 0x7f000789 = 2130708361
I/ACodec: setupAVCEncoderParameters with [profile: Baseline] [level: Level52]
I/ACodec: [OMX.IMG.TOPAZ.VIDEO.Encoder] cannot encode color aspects. Ignoring.
I/ACodec: [OMX.IMG.TOPAZ.VIDEO.Encoder] cannot encode HDR static metadata. Ignoring.
I/ACodec: setupVideoEncoder succeeded
W/ACodec: do not know color format 0x7f000789 = 2130708361
I/AudioRecordPermission: remindWithResult:false
I/HwAudioRecordImpl: HwAudioRecordImpl
I/HwAudioRecordImpl: checkRecordActive
I/HwAudioRecordImpl: sendStateChangedIntent, state=3
E/HW-JPEG-DEC: [HME_JPEG_DEC_Delete](3321): HME_JPEG_DEC_Delete: decoder_ctx=null
E/HW-JPEG-DEC: [HME_JPEG_DEC_Delete](3321): HME_JPEG_DEC_Delete: decoder_ctx=null
I/HwPointEventFilter: do not support AFT because of no config

                      [ 04-08 23:22:44.510   544:31475 E/         ]
                      void hw::COutMetaSplitBase::dump_thermal_meter(monitor_t): open  failed!
I/art: Do partial code cache collection, code=58KB, data=60KB
I/art: After code cache collection, code=57KB, data=60KB
I/art: Increasing code cache capacity to 256KB

       [ 04-08 23:22:47.158   544:31475 E/         ]
       void hw::COutMetaSplitBase::dump_thermal_meter(monitor_t): open  failed!
I/HwPointEventFilter: do not support AFT because of no config
I/art: Do full code cache collection, code=122KB, data=122KB
I/art: After code cache collection, code=107KB, data=79KB

       [ 04-08 23:23:02.321   544:31475 E/         ]
       void hw::COutMetaSplitBase::dump_thermal_meter(monitor_t): open  failed!
I/art: Do partial code cache collection, code=112KB, data=99KB
I/art: After code cache collection, code=112KB, data=99KB
I/art: Increasing code cache capacity to 512KB
`

I think that the Android 7 doesn't support AFT.
 Can you help me?

**Comment by TakuSemba** [comment]
Hi!, your app is not crashing on other devices?
Does it crash only on Huawei devices?

**Comment by camillo860** [comment]
Yes only this device. I resolve with include your project in my application.

**Comment by AmrWatad** [comment]
I have the same problem
do you solved the problem؟

**Comment by ahmedtharwat2** [comment]
Any update for this issue?

**Comment by devtech11** [comment]
i have the same problem on huawei P9 Lite !!!
please did you resolve this problem ?

**Comment by MstrZeroOne** [comment]
I Have the same problem on Huewei Y7 prime.

**Comment by virajtalaviya** [comment]
Same problem on Honor 8

**Comment by virajtalaviya** [comment]
> Yes only this device. I resolve with include your project in my application.

How?
